src/SFA.DAS.AssessorService.Application/Handlers/Search/SearchHandler.cs

```diff
@@ -111,10 +110,12 @@ private async Task<List<SearchResult>> Search(SearchQuery request, CancellationT
 
         private async Task<List<int>> GetEpaoStandards(Organisation thisEpao)
```

Comment:
Can we rename the method? `GetApprovedStandardsForEpao`? `GetEpaoApprovedStandardsWithAtLeastOneVersion` ? I'm trying to make it more descriptive to say what it's doing, GetEpaoStandards does say what it's doing but not the whole story.